src/System.Management.Automation/engine/parser/DebugViewWriter.cs

```diff
@@ -312,12 +322,14 @@ private enum Flow {
             Out(close.ToString(), Flow.Break);
         }
 
+#if !CORECLR
```

Comment:
Dynamic is used in a few places in the dynamic sites (nested dynamic sites!) - so it'd be nice to figure out the right CORECLR api.

Of course, as this is for debugging only, maybe I should have just used reflection instead of copying the DLR code, though we might have made small changes, I don't recall.